src/components/dropdown.tsx

```
@@ -0,0 +1,42 @@
+// 예) <Dropdown size="md" items={[ { label: "수정하기", onClick: handleEdit }, { label: "삭제하기", onClick: handleDelete }, ]} />
+
+interface DropdownProps {
+  size?: "sm" | "md";
+  items: { label: string; onClick: () => void }[];
+}
+
+const Dropdown = ({ size = "md", items }: DropdownProps) => {
+  return (
+    <div
+      className={`absolute ${
+        size === "sm" ? "w-[81px] h-auto" : "w-[106px] h-auto"
```

ESLint

[error] Replace `w-[81px]·h-auto"·:·"w-[106px]·h-auto` with `h-auto·w-[81px]"·:·"h-auto·w-[106px]`

(prettier/prettier)